p2plending/frontend/src/components/LanguageItem.js

```diff
@@ -10,19 +10,40 @@ class LanguageItem extends Component {
     code: PropTypes.string.isRequired,
   }
 
+  getFlagEmoji(lang) {
+    const emojiPath = "https://github.githubassets.com/images/icons/emoji/unicode/";
+    switch(lang) {
+      case 'English':
+        return `${emojiPath}1f1ec-1f1e7.png`
+      case 'Spanish':
+        return `${emojiPath}1f1ea-1f1f8.png`
+      case 'Chinese':
+        return `${emojiPath}1f1e8-1f1f3.png`
+      case 'Korean':
+        return `${emojiPath}1f1f0-1f1f7.png`
+      case 'Japanese':
+        return `${emojiPath}1f1ef-1f1f5.png`
+      case 'Russian':
+        return `${emojiPath}1f1f7-1f1fa.png?` 
+      default:
+        return '⚈';
```

Comment:
This line doesn't work, because it would put the period emoji in where the src is.

Reply:
Additional note, I would suggest we make this a "dict" mapping, not a case statement. Being declarative makes it easier to use in different circumstances.

e.g. 
const LANGUAGE_ICONS = {
  "ru": `${emojiPath}1f1f7-1f1fa.png?`
}

then you can do in your getFlagEmoji function:

return LANGUAGE_ICONS[language_code];